src/protocolgame.cpp

```diff
@@ -415,6 +415,8 @@ void ProtocolGame::parsePacket(NetworkMessage& msg)
 		}
 	}
 
+	addGameTask(&Game::playerExecuteParsePacketEvent, player->getID(), recvbyte, new NetworkMessage(msg));
```

Comment:
this should be on default-switch, uhm? else you will handle twice (c++ and lua). I'm just saying...

Reply:
yeah, we are copying msg here so we dont care. Lua gets his own copy and c++ has its own copy, so we it interfere with each other